### PR TITLE
[Translation zh-tw] _footer.yaml

### PR DESCRIPTION
**Pull request by agektmr**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/344)

### CODE REVIEW

**Comment by agektmr** [comment]
Reviewed by @limhenry translated by @kennith

**Comment by agektmr** [comment]
This is ready to be pulled once CLA issue is cleared.

**Comment by sotayamashita** [comment]
We have signed Corporate CLA but it will takes time to accept a few days according FAQ for Google CLA. https://cla.developers.google.com/about

File: src/content/zh-tw/_footer.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
```
Comment (petele) [inline src/content/zh-tw/_footer.yaml]: I don't think this is in the original, please remove

File: src/content/zh-tw/_footer.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    name: 關鍵主題
+    icon_name: star
+  - contents:
+    - label: 教練帶領培訓
+      path: "/web/ilt/"
+    - label: PWA培訓機構
+      path: "/web/events/pwa-agency-training/"
+    - label: 在Udacity的網絡課程
+      path: "/web/shows/udacity/"
+    icon_name: 學校
```
Comment (petele) [inline src/content/zh-tw/_footer.yaml]: icon_names shouldn't be translated

Comment (sotayamashita) [reply src/content/zh-tw/_footer.yaml]: @kennith Could you modify it?

Comment (kennith) [reply src/content/zh-tw/_footer.yaml]: @sotayamashita Should I change in GitHub or GitLocalize?

Comment (agektmr) [reply src/content/zh-tw/_footer.yaml]: @kennith It's kind of awkward but send a PR to this repository: https://github.com/agektmr/WebFundamentals/tree/gitlocalize-328

Comment (kennith) [reply src/content/zh-tw/_footer.yaml]: @agektmr done. Please review.

**Comment by agektmr** [comment]
Merged @kennith 's fix. cc: @petele

File: src/content/zh-tw/_footer.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  - description: 在Twitter上與@ChromiumDev聯繫。
+    label: 推特
+    icon: "/site-assets/logo-twitter.svg"
+    path: https://twitter.com/chromiumdev
+  - description: 看看我們的視頻。
+    icon: https://www.gstatic.com/images/icons/material/product/2x/youtube_64dp.png
+    label: 視頻
+    path: "/web/shows/"
+  - description: 參加開發者活動。
+    icon_name: 活動
```
Comment (petele) [inline src/content/zh-tw/_footer.yaml]: icon_name should not be translated

**Comment by sotayamashita** [comment]
@googlebot signed

**Pull request merged**